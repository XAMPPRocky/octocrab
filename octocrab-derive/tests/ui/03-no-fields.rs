#![allow(dead_code)]

#[derive(octocrab_derive::Builder)]
struct Test<'a>(&'a str);

fn main() {}
