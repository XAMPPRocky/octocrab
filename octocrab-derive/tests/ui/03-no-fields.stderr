error: only named fields supported
 --> $DIR/03-no-fields.rs:4:16
  |
4 | struct Test<'a>(&'a str);
  |                ^^^^^^^^^
